include/TPP/Dialect/VNNI/VNNIInterfaces.td

```diff
@@ -0,0 +1,86 @@
+#ifndef VNNI_IR_VNNIINTERFACES
```

Comment:
this is new? Look to me I already review this code.

Reply:
The patch is rebased on top of the vnni to tpp code, you'll probably see the same changes here. Once we merge the previous patch, we can review this better.